logic/interfuse/odmr_counter_microwave_interfuse.py

```diff
@@ -44,6 +44,11 @@ class ODMRCounterMicrowaveInterfuse(GenericLogic, ODMRCounterInterface,
     slowcounter = Connector(interface='SlowCounterInterface')
     microwave = Connector(interface='MicrowaveInterface')
 
+    _pulse_out_channel = 'dummy'
```

Comment:
No class variables, either ConfigOption or StatusVar or think about not storing this in the hardware or interfuse module.

Reply:
deleted, as the abstractmethods only have a pass.